netlify.toml

```diff
@@ -1,15 +1,16 @@
 [build]
 publish = "public"
-command = "make production-build"
+command = "git submodule update --init --recursive --depth 1 && make production-build"
 
 [build.environment]
 HUGO_VERSION = "0.81.0"
+NODE_VERSION = "10.19.0"
```

Comment:
This is the solution to our build problems. Had to tell Netlify to use a more modern version of node. Likely we want to bump this to a higher version still... but for now, yay deploy previews!

Reply:
With #133, this added config will no longer be necessary (FYI).

Reply:
Agreed, we should make a note in our todo list to remove it